cpu/esp32/periph/spi.c

```diff
@@ -253,7 +253,7 @@ void IRAM_ATTR spi_acquire(spi_t bus, spi_cs_t cs, spi_mode_t mode, spi_clk_t cl
 
         if (apb_clk / 5 < clk) {
             LOG_TAG_ERROR("spi", "APB clock rate (%"PRIu32" Hz) has to be at "
-                          "least 5 times SPI clock rate (%"PRIu32" Hz)\n",
+                          "least 5 times SPI clock rate (%d Hz)\n",
```

Comment:
Rebasing artifact?

Reply:
The change was by intention. With the change you suggested in https://github.com/RIOT-OS/RIOT/pull/18281#discussion_r920464564, parameter `clk` became enumeration type https://github.com/RIOT-OS/RIOT/blob/f375856a7e2d612c0a8f38052fded69d797cd93b/cpu/esp32/include/periph_cpu.h#L576-L586 which is handled as integer. This led to compilation error on ESP32-C3.